spec.html

```diff
@@ -29418,7 +29418,7 @@
         </emu-note>
         <emu-grammar>CharacterEscape :: HexEscapeSequence</emu-grammar>
         <emu-alg>
-          1. Return the code point value of the SV of |HexEscapeSequence|.
+          1. Return the numeric value of the code unit that is the SV of |HexEscapeSequence|.
```

Comment:
I think I picked `code point value`, so I could use the same phrase within the complete section ([21.2.1.4 Static Semantics: CharacterValue](https://tc39.github.io/ecma262/#sec-patterns-static-semantics-character-value)). Only changing it to `numeric value of the code unit` for `CharacterEscape :: HexEscapeSequence` looks a bit odd, because also other productions in 21.2.1.4 evaluate to a single code unit.

Reply:
> other productions in 21.2.1.4 evaluate to a single code unit.

No, CharacterValue returns an integer in the Unicode codespace, which is too big for a single (16-bit) code unit.

Elsewhere in 21.2.1.4, in occurrences of "the code point value of X" (or "X's code point value"), X either literally denotes a code point (e.g. "U+002D (HYPHEN-MINUS)") or is a metavariable whose value is a code point. But `the SV of |HexEscapeSequence|` is not a code point, so I think it would be misleading to use the same formulation.